tests/damage_rheology_growth.prm

```diff
@@ -0,0 +1,150 @@
+# This is a very simple testcase that checks
+# that the implementation of the grain growth
+# laws are correct. All parameters, including
+# temperature and pressure, are set to constant
+# values and no dynamic recrystallisation is 
+# included (the reciprocal required strain is
+# set to zero). Material flows in from the left
+# and flows out at the right boundary with a constant
+# velocity. The grain size
+# should then only depend on the x coordinate 
+# following the relation
+#
+# d(x) = (d_0 + A * x)^(1/p)
+#
+# with
+# d_0 = 8e-5 m         (initial grain size)
+# p   = 10             (grain size exponent)
+# A   = 2.0294e-46 1/m
+#
+# leading to a grain size of d=8.8957E-5 m at the
+# right boundary of the model. 
```

Comment:
Did you check that? The screen output says 8.072e-05, but I guess you would have to run it for a longer time?

Reply:
Okay, so if you let this model run until and end time of 1.5e6 years, it reaches a steady state of 8.893e-05 (for max composition, which is the grain size in m, and 8.8942e-05 if I increase the global refinements by one). Is that good enough? 
We should probably also mention the end time needed to reach that steady state in the input file, so that future users can reproduce that result more easily.